tests/task/test_eval.py

```diff
@@ -106,12 +107,12 @@ def run_eval_test(params) -> None:
         }
 
     if comet == "skipped":
-        env["COMET_SKIP"] = "1"
+        env["COMET_SKIP"] = "1"  # type: ignore
```

Comment:
Again, the `exclude` paths need updating in the `pyproject.toml`. Rather than fixing files in a big PR, it's better to split them into smaller PRs.